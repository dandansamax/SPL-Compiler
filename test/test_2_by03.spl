
struct Pokemon{
    char name[20];
    int hp;
};

int Hit(struct Pokemon poke){
    poke.hp=poke.hp-5;
    return 0;
}


int test_2_by03(){


    struct Pokemon pokes;


    powerOn(pokes);

    return 0;
}
