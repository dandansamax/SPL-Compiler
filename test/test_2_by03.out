Error type 2 at Line 20: a function is invoked without a definition: powerOn
